model/src/user/current_user_guild.rs

```diff
@@ -0,0 +1,78 @@
+use crate::guild::Permissions;
+use crate::id::GuildId;
+use serde::{Deserialize, Serialize};
+
+/// Information about a guild the current user is in.
+///
+/// This is a partial guild used for the `Get Current User Guilds` endpoint. Refer to the [Discord documentation] for more information.
```

Comment:
Could you wrap this line to 80 characters

Reply:
Addressed